src/proxy/index.js

```diff
@@ -4,7 +4,7 @@ const bodyParser = require('body-parser');
 const routes = require('./routes');
 const config = require('../config');
 const db = require('../db');
-const proxyHttpPort = 8000;
+const proxyHttpPort = process.env.GIT_PROXY_SERVER_PORT || 8000;
```

Comment:
Given there are more than 2 files affected here, it may be better to create a file to setup the environment variables and their defaults to avoid future mistakes

for eg.

```js
// file path: src/common/env.js

const {
  GIT_PROXY_SERVER_PORT = 8000,
  GIT_PROXY_UI_PORT = 8080
} = process.env

export { GIT_PROXY_SERVER_PORT, GIT_PROXY_UI_PORT}
```

then here:

```suggestion
const { GIT_PROXY_SERVER_PORT: proxyHttpPort } = require('../common/env')
```

similar for other files.

this way the defaults are only defined in one placeand easier to maintain.